.github/workflows/tests.yml

```diff
@@ -3,7 +3,7 @@
 
 name: Lint and unittest
 
-on: [push]
+on: [push, pull_request]
```

Comment:
Disclaimer: I'm new to Github Actions.

Would this allow to contributors to run CI in the context of _our_ repository, i.e. gain access to the Ci secrets?

Reply:
Self-answer: no, it wouldn't.

https://docs.github.com/en/actions/reference/events-that-trigger-workflows

> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository. The permissions for the GITHUB_TOKEN in forked repositories is read-only.

Not a problem then.